src/main/osd/osd.h

```diff
@@ -197,6 +198,7 @@ typedef enum {
     OSD_STAT_FLIGHT_DISTANCE,
     OSD_STAT_MAX_FFT,
     OSD_STAT_TOTAL_FLIGHTS,
+    OSD_STAT_TOTAL_PACKS,
```

Comment:
// *** IMPORTANT ***
// DO NOT REORDER THE STATS ENUMERATION. The order here corresponds to the enabled flag bit position
// storage and changing the order will corrupt user settings. Any new stats MUST be added to the end
// just before the OSD_STAT_COUNT entry. YOU MUST ALSO add the new stat to the
// osdStatsDisplayOrder array in osd.c.
//
// IF YOU WANT TO REORDER THE STATS DISPLAY, then adjust the ordering of the osdStatsDisplayOrder array